integration_tests/packages.yml

```diff
@@ -1,2 +1,11 @@
 packages:
-  - local: ../
\ No newline at end of file
+  - package: fivetran/twitter_ads
+    version: [">=0.6.0", "<0.7.0"]
+  - package: fivetran/zendesk
+    version: [">=0.10.0", "<0.11.0"]
+  - package: fivetran/hubspot
+    version: [">=0.8.0", "<0.9.0"]
+  - package: fivetran/jira
+    version: [">=0.11.0", "<0.12.0"]
+  - package: fivetran/klaviyo
+    version: [">=0.5.0", "<0.6.0"]
```

Comment:
Should we maybe make these ranges larger? I would want to make sure we are testing on the latest versions of them.